composer.json

```diff
@@ -16,6 +16,7 @@
         "php": ">=7.1",
 
         "doctrine/inflector": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.3",
```

Comment:
does api platform not work without phpcr? i fear that this is an "optional dependency". would it be possible to do a api-platform/phpcr integration package instead?

Reply:
mh maybe at the end. but i do not know if we want to move this bundle in the same virtual packagage situation, we are, shall we?